### PR TITLE
Mobile update

### PR DESCRIPTION
**Pull request by flaminggenius**
Removed borders around btns also added in dark mode setting
<a href='#crh-start'></a><a href='#crh-data-%7B%22approved%22%3A%20%7B%22https%3A//github.com/flaminggenius%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/14892313%3Fv%3D3%22%7D%7D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/huytd/agar.io-clone/pull/470%23issuecomment-262836405%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%3Ashipit%3A%22%2C%20%22created_at%22%3A%20%222016-11-24T19%3A39%3A55Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/14892313%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/flaminggenius%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3Ashipit%3A%22%2C%20%22created_at%22%3A%20%222016-11-24T19%3A39%3A57Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/14892313%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/flaminggenius%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3Ashipit%3A%22%2C%20%22created_at%22%3A%20%222016-11-24T19%3A40%3A02Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/14892313%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/flaminggenius%22%7D%7D%2C%20%7B%22body%22%3A%20%22Oops%20I%20copied%20the%20files%20over%20from%20my%20server%20where%20I%20was%20working%20on%20it%20my%20bad%20forgot%20to%20change%20it%20back%22%2C%20%22created_at%22%3A%20%222016-11-25T04%3A42%3A14Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/14892313%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/flaminggenius%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%20219ff8310892f738447fa1dc8a757fe5356f3afd%20src/server/server.js%204%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/huytd/agar.io-clone/pull/470%23discussion_r89566291%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Expected%20an%20identifier%20and%20instead%20saw%20%27%3Fname%3D%27.%3Cbr%3E%27%24%27%20is%20not%20defined.%22%2C%20%22created_at%22%3A%20%222016-11-25T05%3A25%3A17Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6697940%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/houndci-bot%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20src/server/server.js%3AL343-350%22%7D%2C%20%22Pull%20e8f5a259c5e9c7ddf77a3657cd02b86ac6bca33c%20src/client/index.html%2017%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/huytd/agar.io-clone/pull/470%23discussion_r89544411%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22This%20shouldn%27t%20be%20here%22%2C%20%22created_at%22%3A%20%222016-11-24T20%3A34%3A17Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/2415829%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/abalabahaha%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20src/client/index.html%3AL26-33%22%7D%2C%20%22Pull%206341fc5f1ccddfd0c0f2905f0cb4adec7a067d2a%20src/server/server.js%2022%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/huytd/agar.io-clone/pull/470%23discussion_r89572562%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%27mysql%27%20is%20not%20defined.%22%2C%20%22created_at%22%3A%20%222016-11-25T07%3A20%3A25Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6697940%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/houndci-bot%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20src/server/server.js%3AL34-56%22%7D%2C%20%22Pull%20075dd09bb1c2e54c741e06d1b4df36738fccc6cb%20src/server/server.js%208%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/huytd/agar.io-clone/pull/470%23discussion_r89568757%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%27XMLHttpRequest%27%20is%20not%20defined.%22%2C%20%22created_at%22%3A%20%222016-11-25T06%3A12%3A56Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6697940%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/houndci-bot%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20src/server/server.js%3AL340-357%22%7D%7D%2C%20%22processed%22%3A%20%5B%22https%3A//github.com/huytd/agar.io-clone/pull/470%23issuecomment-262836405%22%2C%20%22https%3A//github.com/huytd/agar.io-clone/pull/470%23issuecomment-262836408%22%2C%20%22https%3A//github.com/huytd/agar.io-clone/pull/470%23issuecomment-262836417%22%2C%20%22https%3A//github.com/huytd/agar.io-clone/pull/470%23discussion_r89544411%22%2C%20%22https%3A//github.com/huytd/agar.io-clone/pull/470%23issuecomment-262879797%22%2C%20%22https%3A//github.com/huytd/agar.io-clone/pull/470%23discussion_r89566291%22%2C%20%22https%3A//github.com/huytd/agar.io-clone/pull/470%23discussion_r89568757%22%2C%20%22https%3A//github.com/huytd/agar.io-clone/pull/470%23discussion_r89572562%22%5D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>

<a href='https://github.com/flaminggenius'></a>

- [ ] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/huytd/agar.io-clone/pull/470#issuecomment-262836405'>General Comment</a></b>
- <a href='https://github.com/flaminggenius'></a> :shipit:
- <a href='https://github.com/flaminggenius'></a> :shipit:
- <a href='https://github.com/flaminggenius'></a> :shipit:
- <a href='https://github.com/flaminggenius'></a> Oops I copied the files over from my server where I was working on it my bad forgot to change it back
- [ ] <a href='#crh-comment-Pull e8f5a259c5e9c7ddf77a3657cd02b86ac6bca33c src/client/index.html 17'></a> &nbsp;<b><a href='https://github.com/huytd/agar.io-clone/pull/470#discussion_r89544411'>File: src/client/index.html:L26-33</a></b>
- <a href='https://github.com/abalabahaha'></a> This shouldn't be here
- [ ] <a href='#crh-comment-Pull 219ff8310892f738447fa1dc8a757fe5356f3afd src/server/server.js 4'></a> &nbsp;<b><a href='https://github.com/huytd/agar.io-clone/pull/470#discussion_r89566291'>File: src/server/server.js:L343-350</a></b>
- <a href='https://github.com/houndci-bot'></a> Expected an identifier and instead saw '?name='.<br>'$' is not defined.
- [ ] <a href='#crh-comment-Pull 075dd09bb1c2e54c741e06d1b4df36738fccc6cb src/server/server.js 8'></a> &nbsp;<b><a href='https://github.com/huytd/agar.io-clone/pull/470#discussion_r89568757'>File: src/server/server.js:L340-357</a></b>
- <a href='https://github.com/houndci-bot'></a> 'XMLHttpRequest' is not defined.
- [ ] <a href='#crh-comment-Pull 6341fc5f1ccddfd0c0f2905f0cb4adec7a067d2a src/server/server.js 22'></a> &nbsp;<b><a href='https://github.com/huytd/agar.io-clone/pull/470#discussion_r89572562'>File: src/server/server.js:L34-56</a></b>
- <a href='https://github.com/houndci-bot'></a> 'mysql' is not defined.

<a href='https://www.codereviewhub.com/huytd/agar.io-clone/pull/470?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/huytd/agar.io-clone/pull/470?approve=1'></a>&nbsp;<a href='https://www.codereviewhub.com/huytd/agar.io-clone/pull/470?approve=0'></a>&nbsp;<a href='https://github.com/huytd/agar.io-clone/pull/470'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by huytd** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/huytd/agar.io-clone/pull/470'>Refresh</a>.<a href='#crh-update-none'></a>

**Comment by flaminggenius** [comment]
:shipit:

**Comment by flaminggenius** [comment]
:shipit:

**Comment by flaminggenius** [comment]
:shipit:

File: src/client/index.html [hunk 1]
```diff
@@ -26,7 +26,7 @@
     </div>
     <div id="startMenuWrapper">
         <div id="startMenu">
-            <p>Open Agar</p>
+            <p>Mass Warfare</p>
```
Comment (abalabahaha) [inline src/client/index.html]: This shouldn't be here

File: src/client/index.html [hunk 2]
```diff
@@ -57,6 +58,7 @@
         </div>
     </div>
     <!-- JS -->
+  <!--  <script src="virtualjoystick.js"></script>-->
```
Comment (abalabahaha) [inline src/client/index.html]: This shouldn't be here

File: src/client/index.html [hunk 3]
```diff
@@ -1,9 +1,9 @@
-﻿<!doctype html>
+<!doctype html>
 <html lang="en">
 <head>
     <!-- Meta Properties -->
     <meta charset="UTF-8">
-    <title>Agar.IO Clone</title>
+    <title>Mass Warfare</title>
```
Comment (abalabahaha) [inline src/client/index.html]: This shouldn't be here

File: config.json [hunk 4]
```diff
@@ -16,13 +16,13 @@
 	},
     "gameWidth": 5000,
     "gameHeight": 5000,
-    "adminPass": "DEFAULT",
+    "adminPass": "Glitchking12345!#",
```
Comment (abalabahaha) [inline config.json]: The config should be left alone

**Comment by flaminggenius** [comment]
Oops I copied the files over from my server where I was working on it my bad forgot to change it back

**Comment by abalabahaha** [comment]
Seems that you're using the same branch to a few custom mods. I'm going to go ahead and close the PR for now, but feel free to make one with just the proposed changes that are relevant to the main project.

**Pull request closed without merging**